tests/framework/pvc.go

```diff
@@ -0,0 +1,47 @@
+package framework
+
+import (
+	"strings"
+	"time"
+
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+func (f *Framework) CreatePVCFromDefinition(def *k8sv1.PersistentVolumeClaim) (*k8sv1.PersistentVolumeClaim, error) {
```

Comment:
Any desire to have a sister function that takes a `namespace string` parameter for multi namespace tests?

Reply:
As you can see this delegates to a utility function, that utility function will take care of multi NS cases.